app/src/main/java/io/apicurio/registry/storage/impl/sql/RegistryDatasourceProducer.java

```diff
@@ -0,0 +1,77 @@
+package io.apicurio.registry.storage.impl.sql;
+
+import io.agroal.api.AgroalDataSource;
+import io.agroal.api.configuration.supplier.AgroalPropertiesReader;
+import io.apicurio.common.apps.config.Info;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import jakarta.inject.Named;
+import jakarta.ws.rs.Produces;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.slf4j.Logger;
+
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class RegistryDatasourceProducer {
+
+    @Inject
+    Logger log;
+
+    @ConfigProperty(name = "registry.storage.db-kind", defaultValue = "h2")
+    @Info(category = "storage", description = "Datasource Db kind", availableSince = "2.0.0.Final")
+    String databaseType;
+
+    @ConfigProperty(name = "registry.datasource.url", defaultValue = "jdbc:h2:mem:registry_db")
+    @Info(category = "storage", description = "Datasource Db kind", availableSince = "2.0.0.Final")
+    String jdbcUrl;
+
+    @ConfigProperty(name = "registry.datasource.username", defaultValue = "sa")
+    @Info(category = "storage", description = "Datasource Db kind", availableSince = "2.0.0.Final")
+    String username;
+
+    @ConfigProperty(name = "registry.datasource.password", defaultValue = "sa")
+    @Info(category = "storage", description = "Datasource Db kind", availableSince = "2.0.0.Final")
+    String password;
+
+    @ConfigProperty(name = "registry.datasource.jdbc.initial-size", defaultValue = "20")
+    @Info(category = "storage", description = "Datasource Db kind", availableSince = "2.0.0.Final")
+    String initialSize;
+
+    @ConfigProperty(name = "registry.datasource.jdbc.min-size", defaultValue = "20")
+    @Info(category = "storage", description = "Datasource Db kind", availableSince = "2.0.0.Final")
+    String minSize;
+
+    @ConfigProperty(name = "registry.datasource.jdbc.max-size", defaultValue = "100")
+    @Info(category = "storage", description = "Datasource Db kind", availableSince = "2.0.0.Final")
```

Comment:
More duplicate descriptions in the `@Info` annotations.

Reply:
Yup, same as with the others.